src/PSRule.Rules.Azure/Pipeline/Export/ResourceExportVisitor.cs

```diff
@@ -281,6 +281,10 @@ private static async Task<bool> VisitVirtualMachine(ResourceContext context, JOb
                         AddSubResource(resource, await GetResource(context, id, APIVERSION_2022_07_01));
                 }
             }
+
+            var response = await context.GetAsync($"{resourceId}/instanceView", APIVERSION_2021_11_01);
+            resource.Add("PowerState", (string)response["statuses"][1]["code"]);
```

Comment:
Will the status we are after always be at index `1`?

I think we should select first or default with `code` starting with `PowerState/` regardless of index, because it could change at any point.

Something like:

```suggestion
            resource.Add("PowerState", response["statuses"].FirstOrDefault(status => status["code"].Value<string>().StartsWith("PowerState/"))["code"].Value<string>());
```